.gitlab/build-oci.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+source common_build_functions.sh
+
+if [ -n "$CI_COMMIT_TAG" ] && [ -z "$DOTNET_PACKAGE_VERSION" ]; then
+  DOTNET_PACKAGE_VERSION=${CI_COMMIT_TAG##v}
+fi
+
+if [ -z "$ARCH" ]; then
+  ARCH=amd64
+fi
+
+TMP_DIR=$(mktemp --dir)
+
+curl --location --fail \
+  --output $TMP_DIR/datadog-dotnet-apm.old \
+  "https://github.com/DataDog/dd-trace-dotnet/releases/download/v$DOTNET_PACKAGE_VERSION/datadog-dotnet-apm_${DOTNET_PACKAGE_VERSION}_$ARCH.deb"
+
+fpm --input-type deb \
+  --output-type dir \
+  --name datadog-dotnet-apm \
+  --package $TMP_DIR \
+  $TMP_DIR/datadog-dotnet-apm.old
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Can't we re-use the results from the `build` stage directly?

Reply:
No, from what I've been told debs are not built in gitlab but in the azure devops CI

Reply:
> No, from what I've been told debs are not built in gitlab but in the azure devops CI

Out of interest, why does that mean you can't use them directly? (You're correct FWIW)

Reply:
I'm not sure, but looking at `.gitlab/build-deb-rpm.sh`, which builds deb and rpm packages for the injector, I think the azure CI debs are not installed in the expected location. It puts the dotnet package in `/opt/datadog` whereas the injector expects packages to be in `/opt/datadog/apm/libraries`.

Plus I believe we'll do some signing in the future which is required to run on the the internal gitlab CI.